tests/unit-tests/test-class-learner.php

```diff
@@ -20,11 +23,13 @@ public function setUp() {
 
 		$this->factory = new Sensei_Factory();
 
+		self::resetEnrolmentProviders();
 	}//end setUp()
 
 	public function tearDown() {
 		parent::tearDown();
 		$this->factory->tearDown();
+		self::resetEnrolmentProviders();
```

Comment:
It could be only in the `testDeleteUserEnrolments` method. But I already added here for future tests using the providers. Does it make sense?